[2024-02-02 11:16:57,071] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_postgresql.aggregate_dim_table manual__2024-02-02T11:16:34.086943+00:00 [queued]>
[2024-02-02 11:16:57,078] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl_postgresql.aggregate_dim_table manual__2024-02-02T11:16:34.086943+00:00 [queued]>
[2024-02-02 11:16:57,078] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-02-02 11:16:57,078] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2024-02-02 11:16:57,078] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-02-02 11:16:57,086] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): aggregate_dim_table> on 2024-02-02 11:16:34.086943+00:00
[2024-02-02 11:16:57,090] {standard_task_runner.py:52} INFO - Started process 11511 to run task
[2024-02-02 11:16:57,093] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl_postgresql', 'aggregate_dim_table', 'manual__2024-02-02T11:16:34.086943+00:00', '--job-id', '359', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpylksisq8', '--error-file', '/tmp/tmp1f7h73qb']
[2024-02-02 11:16:57,094] {standard_task_runner.py:80} INFO - Job 359: Subtask aggregate_dim_table
[2024-02-02 11:16:57,129] {task_command.py:369} INFO - Running <TaskInstance: etl_postgresql.aggregate_dim_table manual__2024-02-02T11:16:34.086943+00:00 [running]> on host c3bb523480bf
[2024-02-02 11:16:57,177] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl_postgresql
AIRFLOW_CTX_TASK_ID=aggregate_dim_table
AIRFLOW_CTX_EXECUTION_DATE=2024-02-02T11:16:34.086943+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-02-02T11:16:34.086943+00:00
[2024-02-02 11:17:00,036] {logging_mixin.py:115} INFO - Successfully install sqlalchemy
[2024-02-02 11:17:02,910] {logging_mixin.py:115} INFO - Successfully install sqlalchemy
[2024-02-02 11:17:05,859] {logging_mixin.py:115} INFO - Successfully install pymysql
[2024-02-02 11:17:08,786] {logging_mixin.py:115} INFO - Successfully install sqlalchemy
[2024-02-02 11:17:08,861] {python.py:173} INFO - Done. Returned value was: insert dim table
[2024-02-02 11:17:08,887] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_postgresql, task_id=aggregate_dim_table, execution_date=20240202T111634, start_date=20240202T111657, end_date=20240202T111708
[2024-02-02 11:17:08,910] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-02-02 11:17:08,938] {local_task_job.py:273} INFO - 5 downstream tasks scheduled from follow-on schedule check
